File: front-end/src/app/service-models/app/v1/client/client-company-holidays-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';

export class ClientCompanyHolidaysSM extends CoinManagementServiceModelBase<number> {
    name!: string;
    description!: string;
    dateTime!: Date;
    clientCompanyDetailId!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/Client/ClientCompanyHolidaysSM.cs
namespace Siffrum.Web.Payroll.ServiceModels.v1.Client
{
    public class ClientCompanyHolidaysSM : Base.SiffrumPayrollServiceModelBase<int>
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime DateTime { get; set; }
        public int ClientCompanyDetailId { get; set; }
    }
}
